jetstream/desktop-release-show-relay-to-all-browsers.toml

```diff
@@ -6,8 +6,162 @@ description = "User exposed to logins dropdown on a login form"
 data_source = "events_form_autocomplete"
 select_expression = "event_name = 'show_logins'"
 
+[metrics]
+[metrics.relay_integration_shown_relay_offer]
+friendly_name = "Relay Optin: Relay offer shown to client"
+description = "Number of times Relay offer is shown"
+select_expression = """
+      COALESCE(COUNTIF(
+         event_method = 'shown'
+         AND event_object = 'offer_relay'
+      ),0)
+"""
+data_source = "relay_integration_events"
```

Comment:
You can skip the stuff that stays the same, it should still get picked up from the outcome since the metric definitions aren't changing. Not a big deal though if you'd rather be explicit here.